docs/metrics.md

```diff
@@ -144,8 +148,10 @@ Metrics below are captured within load balancer
 
 Metrics below are for invoker state as recorded within load balancer monitoring.
 
-* `openwhisk.counter.loadbalancer_invokerOffline_count` - Records the count of invokers considered offline based on health pings.
-* `openwhisk.counter.loadbalancer_invokerUnhealthy_count` - Records the count of invokers considered unhealthy based on health pings.
+* `openwhisk.counter.loadbalancer_totalHealthyInvoker<actionType>_count` - Records the count of managed invokers considered healthy based on health pings. actionType defines whether it is a managed or a blackbox action
+* `openwhisk.counter.loadbalancer_totalUnresponsiveInvoker<actionType>_count` - Records the count of managed invokers considered unresponsive based on health pings. actionType defines whether it is a managed or a blackbox action
```

Comment:
If I'm not mistaken, invoker state _unresponsive_ does not base on (missing) health pings from invokers but from (missing) activation responses from invoker.

Reply:
I will enhance the statements for all the states to make it clear.